### PR TITLE
chore(babel): update babel configuration for async await

### LINKED ISSUE #23: chore(babel): update babel configuration for async await
* include `babel-polyfill` and `babel-preset-es2015` on master as dev dependencies
* add `es2015` in `presets` array in .babelrc
* add `babel-polyfill` to `entry` key in webpack config file

### PR DESCRIPTION
**Pull request by jluria**
Closes #23 

* include `babel-polyfill` on master as dev dependency
* add `babel-polyfill` to `entry` key in webpack config file

### CODE REVIEW

File: w3/.babelrc [hunk 1]
```diff
@@ -11,7 +11,8 @@
       }
     ],
     // This tells babel we want to use react w/ es6
+    "es2015",
```
Comment (JonARea) [inline w3/.babelrc]: We shouldn't need this if we already have babel-preset-env

File: w3/package.json [hunk 2]
```diff
@@ -27,6 +27,8 @@
   "devDependencies": {
     "babel-core": "^6.26.0",
     "babel-loader": "^7.1.2",
+    "babel-polyfill": "^6.26.0",
+    "babel-preset-es2015": "^6.24.1",
```
Comment (JonARea) [inline w3/package.json]: babel-preset-es2015 is redundant, babel-preset-env is sufficient

**Review by JonARea — CHANGES_REQUESTED** [review]
We shouldn't need babel-preset-2015

File: w3/.babelrc (hunk 1 above)
Comment (jluria) [reply w3/.babelrc]: 👍

File: w3/package.json (hunk 2 above)
Comment (jluria) [reply w3/package.json]: I'll remove

File: w3/.babelrc (hunk 1 above)
Comment (jluria) [reply w3/.babelrc]: That's a good point

**Comment by jluria** [comment]
@danceoval PR has been reviewed and is ready for a merge

**Pull request merged**